ee/clickhouse/errors.py

```diff
@@ -17,6 +21,9 @@ def wrap_query_error(err: Exception) -> Exception:
 
     # :TRICKY: Return a custom class for every code by looking up the short name and creating a class dynamically.
     if hasattr(err, "code"):
+        if err.code == 427:
```

Comment:
Where does this `427` come from? Maybe add a comment on how CH handles this and why we raise it this way?

Reply:
So, this is the error coming from CH, whose name is `CHQueryErrorCannotCompileRegexp` , which occurs when the regex being passed in is invalid.